exe-requirements.txt

```diff
@@ -11,3 +11,5 @@ pyinstaller==4.10; platform.machine == "aarch64"
 # During startup program terminated with signal SIGSEGV, Segmentation fault.
 # staying with 0.12.1 for the mean time...
 staticx==0.12.1; platform.machine == "x86_64"
+netifaces==0.11.0; sys.platform == "win32"
```

Comment:
These (netifaces & WMI) should move to `requirements.txt`. Logically, `requirements.txt` is the general requirements file for gProfiler, and `exe-requirements.txt` is the one for requirements that are only for the pyinstaller build. These 2 libraries are part of gProfiler and not of the exe build.

Reply:
Done